src/wallet.js

```diff
@@ -1,142 +1,141 @@
-const bitcoin = require("bitcoinjs-lib");
+const bitcoin = require('bitcoinjs-lib');
 const blockexplorer = require('blockchain.info/blockexplorer');
 const logger = require('winston');
-logger.level = "debug";
+logger.level = 'debug';
 const apiCall = require('../src/apiCall.js');
 const pushtx = require('blockchain.info/pushtx');
-let calcSize = (Transaction) => {
+
+const calcSize = (Transaction) => {
     const txSize = (Transaction.tx.ins.length * 180 + Transaction.tx.outs.length * 34 + 10 + -Transaction.tx.ins.length);
     return txSize; // Transaction size in Bytes
+};
 
-}
-let calcFee = (Transaction, feeType) => {
-    return new Promise((fulfill, reject) => {
-        const size = calcSize(Transaction);
-        let query = 'https://bitaps.com/api/fee';
-        let feeBase = 0;
-        apiCall.getData(query).then((feeData) => {
-            switch (feeType) {
-                case 'medium':
-                    feeBase = feeData.medium;
-                case 'high':
-                    feeBase = feeData.high;
-                default:
-                    feeBase = feeData.low;
-            }
-            if (size * feeBase < 20000) {  // Minimum relay fee
-                fulfill(20000);
-            }
-            fulfill(size * feeBase);
-        }).catch((err) => {
-            reject(err);
-        });
+const calcFee = async(Transaction, feeType) => {
+    const size = calcSize(Transaction);
+    const query = 'https://bitaps.com/api/fee';
+
+    const feeData = await apiCall.getData(query);
+    const feeBase = feeData[feeType] || feeData.low || 0;
+    
+    if (size * feeBase < 20000) {  // Minimum relay fee
+        return 20000;
+    }
+
+    return size * feeBase;
+};
+
+const pushPay = async(payees, payor, opcode) => {
+    if (!payor.address) {
+        return Promise.reject(
+            new Error('Payor address is required')
+        );
+    }
+
+    if (!opcode) {
+        opcode = 'Chainbyteswallet';
+    }
+
+    const data = Buffer.from(opcode);
```

Comment:
Use `Buffer.from` instead of `new Buffer` which is deprecated and it's a security risk.